WordPress/gutenberg
# [RNMobile] Editor Onboarding: "The Basics" help section - Details Part 1

**Gutenberg-Mobile PR**: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3777

## Description

This PR is the second in a series of PRs to add the Editor Onboarding basic help section defined in this ticket: https://github.com/WordPress/gutenberg/issues/30626 and covers the following:
- Adds the "details" pages for each of the basic help topics including localized text and static images (additional notes below)
Note that this PR only covers a baseline for this area of the feature and will continue to be iterated on by another developer. I'm opening this PR now to ensure the work done so far so it can neatly be handed off to the next developer as I'll be moving projects. 

**Special notes:**
- The images provided are not localized and will need to be replaced with a mechanism for creating localized images.
- Since the images will eventually be replaced before this feature rolls out to production, I have no included different sizes for different screen resolutions. 
- Does not include any considerations for accessibility
- It would make sense to make the detail page full screen in the future if needed.
- For some reason, the images would not appear if the Demo app was built directly in the gutenberg repo, but would appear if built from the gutenberg-mobile repo. @dcalhoun wrote a patch to fix this issue and I have included it in dce0398dbd4869f84d4b8701da08ac568d411e9d. 
- I attempted to use `BottomSheet.SubSheet` for the details pages, but for some reason the images would flicker uncontrollably while scrolling on iOS. I attempted to use `useMemo` to prevent this, but couldn't get it to work. For that reason I went back to my original custom implementation of `BottomSheet.NavigationScreen` and use `useMemo` in `view-section.native.js` when serving up the image assets to avoid rerendering images while scrolling. This works nicely. Below are recordings of the iOS differences. 

The "before" is using `BottomSheet.SubSheet`:

https://user-images.githubusercontent.com/5810477/127734916-90177e66-8c45-4d48-9039-b315245951ba.mp4

After is using the custom `BottomSheet.NavigationSheet`:

https://user-images.githubusercontent.com/5810477/127734945-d82e55bc-136d-4937-8ac9-666bc9ec729e.mp4

## How has this been tested?



1. Build and run the app on iOS and Android
2. Open the menu (shake on Android) and tap on "Help". The Help "Basics" list of topics will be shown.
3. Tap on each top and verify the appropriate bottom sheet is displayed. 

## Screenshots 

https://user-images.githubusercontent.com/5810477/127735228-a36cf2ec-9ab7-420d-865b-a7dcb2b31e0d.mp4



## Types of changes





## Checklist:
